code/modules/mob/living/carbon/human/species/_species.dm

```diff
@@ -224,7 +242,12 @@
 		qdel(ears)
 
 	if(mutantears)
-		ears = new mutantears(H)
+		var/obj/item/organ/internal/ears/new_ears = new mutantears()
```

Comment:
cam you check the `parent_organ` for these before new'ing this organ? Just to avoid having to qdel it immediately after

Reply:
I'm not entirely sure, I think `parent_organ` might need to be static for that?

Reply:
never mind, I think I've got something that works.